server/src/main/java/org/elasticsearch/search/fetch/FetchContext.java

```diff
@@ -77,13 +76,6 @@ public MapperService mapperService() {
         return searchContext.mapperService();
     }
```

Comment:
Can we remove FetchContext#mapperService() as well?

Reply:
not yet, there is some leftover usage around indexAnalyzer that I did not address. Do you have suggestions on how to get rid of that?

Reply:
QueryShardContext has a getIndexAnalyzer() method, you could use that instead I think?

Reply:
yes you are right, it's an easy one to remove, will do that

Reply:
it appears that I forgot to push that one commit to this PR, I opened #64620